### ISSUE TITLE
[BUG] Attention mask and pad token not set

### ISSUE THREAD

**Issue by sild928** [lllyasviel/Omost#20]
`The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.
C:\Users\sildenafil\.conda\envs\omost\lib\site-packages\transformers\models\llama\modeling_llama.py:649: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:455.)
  attn_output = torch.nn.functional.scaled_dot_product_attention(
You shouldn't move a model that is dispatched using accelerate hooks.`

Runs fine it appears, but not sure what this means.

**Comment by Sacredgenius** [comment]
me,too
